### ISSUE TITLE
Install.wim file missing from resulting ISO image

### ISSUE THREAD

**Issue by alphaspeaks** [ntdevlabs/tiny11builder#220]
[tiny11.log](https://github.com/user-attachments/files/16228389/tiny11.log)
Downloaded the latest ISO image from Microsoft: "Win11_23H2_EnglishInternational_x64v2"
Ran tiny11Coremaker - processed with no errors reported.
Ran tiny11maker - processed, again with no errors.
Shoved the ISO in a Virtualbox instance, and booted it, Virtualbox threw an error, citing "Windows could not collect information for  [OSImage] because the specified image file [install.wim] does not exist.

So I pull the ISO image, mount it, and sure enough:
![Screenshot Win11 ISO](https://github.com/user-attachments/assets/a7134828-d1e0-4181-9174-c53859f6c12e)
It's there in the Windows ISO, but....
![Screenshot tiny11 ISO](https://github.com/user-attachments/assets/60d9d2f7-3366-43d5-8f6e-8e0c91b25d61)
Not there in the ISO source file.

I'll try with another version of the ISO file for Windows.

**Comment by Nico-Mayer** [comment]
@alphaspeaks did another iso fixed it for you? I got the exact same issue.

**Comment by alphaspeaks** [comment]
No, I tried using the base win11 one, and it caused more issues.
-------- Original Message --------On 14/08/2024 16:29, Nico Mayer  wrote:
@alphaspeaks did another iso fixed it for you? I got the exact same issue.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/ntdevlabs/tiny11builder/issues/220#issuecomment-2289117608",
"url": "https://github.com/ntdevlabs/tiny11builder/issues/220#issuecomment-2289117608",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by foltz61** [comment]
This happened to my build. I ran it again and it worked.
